src/dotnet-compile-fsc/Program.cs

```diff
@@ -12,15 +12,16 @@
 using Microsoft.DotNet.Cli.Compiler.Common;
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Resolution;
 using NuGet.Frameworks;
+using System.Reflection;
 
 namespace Microsoft.DotNet.Tools.Compiler.Fsc
 {
     public class CompileFscCommand
```

Comment:
This class right now seems super confusing to me. It is doing a bunch of different things inside of it. Things that I would image we should have abstractions for, like Resolving FSC. Can we spend a bit of time refactoring this?